### PR TITLE
Fix 2017 Genesis G80 car harness and shop link in docs

### PR DESCRIPTION
**Pull request by SeanCox**
**Description** [](A description of the bug and the fix. Also link any relevant issues.)

The 2017 Genesis G80 Harness is different than the 2018-19 Harnesses. The 2017 also has the limitations of not being useable under 19mph for long and 37mph for lat. 

Here are some eBay links showing the difference in the lane keep assist cameras between the 2017 and 2018 year models:

2017:
https://www.ebay.com/itm/314008788332?chn=ps&_trkparms=ispr%3D1&amdata=enc%3A1qV1mvaERRaexn3OSGvsi5Q27&norover=1&mkevt=1&mkrid=711-117182-37290-0&mkcid=2&mkscid=101&itemid=314008788332&targetid=1644837434603&device=c&mktype=&googleloc=9011539&poi=&campaignid=16743749222&mkgroupid=138744546207&rlsatarget=pla-1644837434603&abcId=9300842&merchantid=7866994&gclid=Cj0KCQiAq5meBhCyARIsAJrtdr7XJYQKoOQFCMd-X7fcQNxz7Dph92aNE9KbcpgsDUjzqk6XCNASbgkaAnQCEALw_wcB

2018:
https://www.ebay.com/itm/265208107107?chn=ps&norover=1&mkevt=1&mkrid=711-117182-37290-0&mkcid=2&mkscid=101&itemid=265208107107&targetid=1645685073288&device=c&mktype=&googleloc=9011539&poi=&campaignid=16730423415&mkgroupid=135815925780&rlsatarget=pla-1645685073288&abcId=9300841&merchantid=101677473&gclid=Cj0KCQiAq5meBhCyARIsAJrtdr58z_q80cN2pr-GUm8nSa6bkIQg_9QX88OO96E6Imf5CJ1G2U58LvYaAgWVEALw_wcB

**Verification** [](Explain how you tested this bug fix.)

I bought a 2017 Genesis G80 and the comma 3 with the H harness only to find out I needed a J harness and the limitations my car has.

I have also talked to 2 other discord users that have 2017 G80s that used the J harness.

### CODE REVIEW

File: docs/CARS.md [hunk 1]
```diff
@@ -30,7 +30,8 @@ A supported vehicle is one that just works when you install a comma three. All s
 |comma|body|All|openpilot|0 mph|0 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|None||
 |Genesis|G70 2018-19|All|Stock|0 mph|0 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|<a href="https://comma.ai/shop/comma-three.html?make=Genesis&model=G70 2018-19">Hyundai F</a>||
 |Genesis|G70 2020|All|openpilot available[<sup>1</sup>](#footnotes)|0 mph|0 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|<a href="https://comma.ai/shop/comma-three.html?make=Genesis&model=G70 2020">Hyundai F</a>||
-|Genesis|G80 2017-19|All|Stock|0 mph|0 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|<a href="https://comma.ai/shop/comma-three.html?make=Genesis&model=G80 2017-19">Hyundai H</a>||
+|Genesis|G80 2017|Smart Cruise Control (SCC)|Stock|19 mph|37 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|<a href="https://comma.ai/shop/comma-three.html?make=Hyundai&model=Genesis 2015-16">Hyundai J</a>||
```
Comment (SeanCox) [inline docs/CARS.md:33]: I am using the store link to the  2015-16 Hyundai Genesis since that is the same car with the correct harness.

**Comment by sshane** [comment]
Can you edit [selfdrive/car/hyundai/values.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/hyundai/values.py) instead? I can re-generate

**Comment by SeanCox** [comment]
> Can you edit [selfdrive/car/hyundai/values.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/hyundai/values.py) instead? I can re-generate

Sure thing. I'll get started on it now.

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -280,7 +282,22 @@ class Buttons:
   CAR.KIA_STINGER: [{
     67: 8, 127: 8, 304: 8, 320: 8, 339: 8, 356: 4, 358: 6, 359: 8, 544: 8, 576: 8, 593: 8, 608: 8, 688: 5, 809: 8, 832: 8, 854: 7, 870: 7, 871: 8, 872: 8, 897: 8, 902: 8, 909: 8, 916: 8, 1040: 8, 1042: 8, 1056: 8, 1057: 8, 1064: 8, 1078: 4, 1107: 5, 1136: 8, 1151: 6, 1168: 7, 1170: 8, 1173: 8, 1184: 8, 1265: 4, 1280: 1, 1281: 4, 1287: 4, 1290: 8, 1292: 8, 1294: 8, 1312: 8, 1322: 8, 1342: 6, 1345: 8, 1348: 8, 1363: 8, 1369: 8, 1371: 8, 1378: 4, 1379: 8, 1384: 8, 1407: 8, 1419: 8, 1425: 2, 1427: 6, 1456: 4, 1470: 8
   }],
-  CAR.GENESIS_G80: [{
+    CAR.GENESIS_G80: [{
```
Comment (SeanCox) [inline selfdrive/car/hyundai/values.py:285]: @sshane This is a copy of the HYUNDAI_GENESIS fingerprint. Just want to make sure this is the correct way to do this.

**Comment by sshane** [comment]
Can you share your dongle ID?

**Comment by sshane** [comment]
You had the right idea, but you probably just didn't know that you can have two CarInfo objects per openpilot platform. To generate the CARS.md I ran `selfdrive/car/docs.py`

Fixed it for you, do the fields look right for you car?

**Comment by sshane** [comment]
The FW from this 2017 Genesis G80 indeed matches the Hyundai Genesis's format: https://github.com/commaai/openpilot/pull/27171

**Pull request merged**